### PR TITLE
add color picker to home page, minor styling adjustments

### PR DESCRIPTION
**Pull request by jsheng112**
Two color pickers for background color and text color are added to the index.html page, and javascript functions to support these are added as well. Minor css changes to layout and blog posts.

### CODE REVIEW

**Comment by meltsufin** [comment]
@jsheng112 Notice that these changes affect other pages as well and may not work there. For example, on the Experience page, you have these divs with their own fixed background color defined by the `.experience` style, and some text colors that work on the user-defined background may not work well on that fixed background color of the divs on the other pages.

**Comment by jsheng112** [comment]
@meltsufin Thanks for reviewing my code! Affecting the other pages was intentional. It originally did not but I was reading about the javascript local storage and wanted to use it somehow. The whole color picker probably does not have a place in a personal website and I will eventually remove it later on or move it to a "settings" tab (it was more of an exercise for me to see how to use javascript to change css).

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -127,4 +126,25 @@ body {
...
+  float: left;
+  width: 45%;
+  padding: 15px;
+}
+
+#color {
+  margin-bottom: 1em;
+}
+
+#color-text {
```
Comment (elefeint) [inline portfolio/src/main/webapp/style.css:148]: You can apply the same style to multiple unrelated tag hierarchies:
```
#color, #color-text {
  margin-bottom: 1em; 
}
```

**Review by elefeint — APPROVED** [review]
Looks awesome! It's fun to put together color combinations pleasing to the eye.

**Review by meltsufin — APPROVED** [review]
Local storage is cool feature, and yes, it's good to experiment.

**Pull request merged**